distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Table_Helpers.enso

```diff
@@ -1,4 +1,5 @@
 from Standard.Base import all
+import Standard.Base.Data.Text.Regex.Regex
```

Comment:
Not needed as in `from Standard.Base import all`.

Reply:
Done